test/test_configuration.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021- Python Language Server Contributors.
+
+import os
+from unittest.mock import patch
+
+from test.fixtures import CALL_TIMEOUT_IN_SECONDS
+from test.test_notebook_document import wait_for_condition
+
+import pytest
+
+from pylsp import IS_WIN
+
+
+INITIALIZATION_OPTIONS = {
```

Comment:
Using this as an example as this is a real world example for when to override confs